src/Integration.Vsix.UnitTests/CFamily/CFamilyHelperTests.cs

```diff
@@ -523,8 +527,8 @@ public void CreateRequest_ErrorGetting_IsHandled()
             var loggerMock = new Mock<ILogger>();
 
             var projectItemMock = CreateProjectItemWithProject("c:\\foo\\xxx.vcxproj");
-            var vcProjectMock = projectItemMock.Object.Object as VCProjectMock;
-            vcProjectMock.ActiveConfiguration = null;
+            // NB we want the exception to be thrown from inside the FileConfig::TryGet
```

Comment:
I think it's better to rename this test instead of the comment, e.g. 
CreateRequest_ExceptionInFileConfig_IsHandled